plugin/trino-mariadb/src/test/java/io/trino/plugin/mariadb/TestMariaDbTypeMapping.java

```diff
@@ -812,6 +813,76 @@ private void testTimestamp(ZoneId sessionZone)
                 .execute(getQueryRunner(), session, trinoCreateAndInsert("test_timestamp"));
     }
 
+    @Test
+    public void testDatetime()
```

Comment:
The current place is between `testTimestamp` and `testIncorrectTimestamp`. Move this test under `testTimestamp` and add `testIncorrectDatetime` test.